chapter2/logger.md

```diff
@@ -33,5 +33,5 @@ config :membrane_core, Membrane.Logger,
 
 You have to make sure, that logger module is available in your application. To use `Membrane.Loggers.Console`, you have to add it as a dependency to your `mix.exs`:
 ```elixir
-{:membrane_loggers, git: "git@github.com:membraneframework/membrane-loggers.git"}
+{:membrane_loggers, "~> 0.1"}
```

Comment:
This will probably change before release (we need to enforce compatibility with core 0.2)